python/lsst/pipe/base/executionButlerBuilder.py

```diff
@@ -240,7 +240,7 @@ def _export(butler: Butler, collections: Iterable[str] | None, inserts: DataSetT
     # export/import
     BackendClass = get_class_of(butler._config["repo_transfer_formats", "yaml", "export"])
     backend = BackendClass(yamlBuffer, universe=butler.dimensions)
-    exporter = RepoExportContext(butler.registry, butler._datastore, backend, directory=None, transfer=None)
+    exporter = RepoExportContext(butler._registry, butler._datastore, backend, directory=None, transfer=None)
```

Comment:
It is looking suspiciously like `RepoExportContext` is not a public API...

Reply:
I think its instantiation is intended to happen inside `Butler.export()`, but execution butler does not like to use export() for some reason. I prefer to wait until we remove execution butler rater than fixing it.

Reply:
I consider everything but construction of `RepoExportContext` to be public API, but construction definitely is not.